apps/console/src/features/organizations/api/organization-role.ts

```diff
@@ -210,3 +210,156 @@ export const getOrganizationRoleById = (organizationId: string, roleId: string):
             return Promise.reject(error);
         });
 };
+
+/**
+ * Mock getter for Organization Permission List
+ */
+export const getOrganizationPermissions = (): Promise<any> => {
```

Comment:
Why do we need this?

Reply:
For now, we don't have a BE endpoint to get the organization permissions. Since that is available, decided to hardcode the permission list in the FE as a mock

Reply:
Added a descriptive comment in https://github.com/wso2/identity-apps/pull/3153/commits/86522a5a7540ca5b5f8245a95c5a59e9f02a4017